endpoint_cnt.go

```diff
@@ -113,13 +113,15 @@ func (ec *endpointCnt) updateStore() error {
 	if store == nil {
 		return fmt.Errorf("store not found for scope %s on endpoint count update", ec.DataScope())
 	}
+	count := ec.EndpointCnt()
```

Comment:
@aboch  Yes, we should also make a copy of the network pointer. 
In my case, the cache object is already initialized in https://github.com/docker/libnetwork/blob/master/endpoint.go#L1102 on clean up local endpoints. After debugging more, I found the value `18446744073709551593` is not a random value, it's a overflowed value caused by https://github.com/docker/libnetwork/blob/master/endpoint_cnt.go#L139.
during local endpoints cleaning up, the value was decline to 0. we should also fix this.